server/src/repository/AccountsDAO.js

```diff
@@ -0,0 +1,49 @@
+const {DynamoDBClient} = require('@aws-sdk/client-dynamodb');
+const {DynamoDBDocumentClient, PutCommand, QueryCommand} = require('@aws-sdk/lib-dynamodb');
+const {fromIni} = require('@aws-sdk/credential-provider-ini');
+
+const client = new DynamoDBClient({
+    region: 'us-west-1',
+    credentials: fromIni({profile: 'cb_account'})
+});
+const documentClient = DynamoDBDocumentClient.from(client);
+const TableName = 'CookedBook-Accounts';
+
+async function createNewAccount(Item) {
+    const command = new PutCommand({
+        TableName,
+        Item
+    });
+
+    try {
+        const data = await documentClient.send(command);
+        return data;
+    } catch (error) {
+        // logger.error(error);
+        console.error(error);
+    }
+    return null;
+}
+
+async function getAccountByUsername(username) {
+    const command = new QueryCommand({
```

Comment:
Since our `Accounts` table only has a partition key, a `GetCommand` would be better.